types/algoliasearch/index.d.ts

```diff
@@ -1880,9 +1881,254 @@ declare namespace algoliasearch {
   }
 }
 
-declare function algoliasearch(
-  applicationId: string,
-  apiKey: string,
-  options?: algoliasearch.ClientOptions
-): algoliasearch.Client;
+declare namespace algoliasearchInitPlaces {
```

Comment:
Where does that name come from? I couldn't find it in the source of `places` package (see https://github.com/algolia/places/search?q=algoliasearchInitPlaces&unscoped_q=algoliasearchInitPlaces).

I'm not really fond of that name :) If we could change it, I'd highly appreciate changing it to something like `algoliasearch.Places`. However, if it's a requirement to augment a specific JavaScript module that I can't find, then we have to roll with it I guess.

Reply:
Could you take a look, i made some changes 😁

Reply:
Perfect! Thank you very much 🥰